lib/generators/sufia/install_generator.rb

```diff
@@ -44,22 +37,7 @@ def run_curation_concerns_work_generator
 
     # Setup the database migrations
     def copy_migrations
-      [
-        "acts_as_follower_migration.rb",
-        "add_social_to_users.rb",
-        "add_ldap_attrs_to_user.rb",
-        "add_avatars_to_users.rb",
-        "create_local_authorities.rb",
-        "create_trophies.rb",
-        'add_linkedin_to_users.rb',
-        'create_tinymce_assets.rb',
-        'create_content_blocks.rb',
-        'create_featured_works.rb',
-        'add_external_key_to_content_blocks.rb',
-        'create_uploaded_files.rb'
-      ].each do |file|
-        better_migration_template file
-      end
+      rake 'sufia:install:migrations'
```

Comment:
@jcoyne But our classes are not timestamped before the migrations are generated.  The order is guaranteed once the app has been generated but what order do the migrations get created in?